site-config/lungmap/dev/detail/project/mainColumn.ts

```diff
@@ -2,30 +2,79 @@ import {
+import * as V from "../../../../../app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders";
 
-// Data Release Policy
-const DATA_RELEASE_POLICY: ComponentConfig = {
-  children: [
-    {
-      component: MDX.DataReleasePolicy,
-    } as ComponentConfig<typeof MDX.DataReleasePolicy>,
-  ],
-  component: C.CollapsableSection,
-  props: {
-    collapsable: false,
-    title: "Data Access Policy",
-  },
-};
-
-/**
- * Returns overview main column components.
- * @returns overview main column components.
- */
-export function getOverviewMainColumn(): ComponentsConfig {
-  const mainColumn = [...hcaOverviewMainColumn];
-  mainColumn.pop(); // Remove the last element (HCA Data Release Policy).
-  return [...mainColumn, DATA_RELEASE_POLICY];
-}
+export const mainColumn: ComponentsConfig = [
+  {
+    component: C.Description,
+    viewBuilder: V.buildDescription,
+  } as ComponentConfig<typeof C.Description, ProjectsResponse>,
+  {
+    component: C.Contacts,
+    viewBuilder: V.buildContacts,
+  } as ComponentConfig<typeof C.Contacts, ProjectsResponse>,
+  {
+    component: C.Publications,
+    viewBuilder: V.buildPublications,
+  } as ComponentConfig<typeof C.Publications, ProjectsResponse>,
+  {
+    component: C.Contributors,
+    viewBuilder: V.buildContributors,
+  } as ComponentConfig<typeof C.Contributors, ProjectsResponse>,
+  {
+    component: C.CollaboratingOrganizations,
+    viewBuilder: V.buildCollaboratingOrganizations,
+  } as ComponentConfig<typeof C.CollaboratingOrganizations, ProjectsResponse>,
+  {
+    component: C.DataCurators,
+    viewBuilder: V.buildDataCurators,
+  } as ComponentConfig<typeof C.DataCurators, ProjectsResponse>,
+  {
+    component: C.Citation,
+    viewBuilder: V.buildCitation,
+  } as ComponentConfig<typeof C.Citation, ProjectsResponse>,
+  {
+    component: C.SupplementaryLinks,
+    viewBuilder: V.buildSupplementaryLinks,
+  } as ComponentConfig<typeof C.SupplementaryLinks>,
+  {
+    children: [
+      {
+        component: C.KeyValuePairs,
+        viewBuilder: V.buildAccessions,
+      } as ComponentConfig<typeof C.KeyValuePairs, ProjectsResponse>,
+    ],
+    component: C.CollapsableSection,
+    props: {
+      collapsable: true,
+      title: "Accessions",
+    },
+  } as ComponentConfig<typeof C.CollapsableSection>,
+  {
+    children: [
+      {
+        component: MDX.DataReleasePolicy,
+      } as ComponentConfig<typeof MDX.DataReleasePolicy>,
+    ],
+    component: C.CollapsableSection,
+    props: {
+      collapsable: false,
+      title: "Data Access Policy",
+    },
+  } as ComponentConfig<typeof C.CollapsableSection>,
+  {
+    children: [
+      {
+        component: MDX.DataReleasePolicy,
+      } as ComponentConfig<typeof MDX.DataReleasePolicy>,
+    ],
+    component: C.CollapsableSection,
+    props: {
+      collapsable: false,
+      title: "Data Access Policy",
+    },
```

Comment:
It looks like the `Data Access Policy` config is duped here:

<img width="877" alt="Screenshot 2024-12-02 at 11 14 31 AM" src="https://github.com/user-attachments/assets/9069c557-694e-4f72-9697-0b1f85db948e">

Reply:
@MillenniumFalconMechanic whoops - should be good now! Thanks!